pkgs/development/python-modules/dockerfile-parse/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv, buildPythonPackage, fetchPypi, six, pytestcov, pytest }:
+
+buildPythonPackage rec {
+  version = "0.0.13";
+  pname = "dockerfile-parse";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1p0x81q3m3nlj4rqal9a959xcbjhncb548wd4wr3l7dpiajqqc9c";
+  };
+
+  patchPhase = ''
+    echo " " > tests/requirements.txt \
+  '';
+
+  propagatedBuildInputs = [ six ] ;
+
+  checkInputs = [ pytestcov pytest ];
+
+  meta = with stdenv.lib; {
+    description = "Python library for parsing Dockerfile files";
```

```suggestion
    description = "Python library for parsing Dockerfiles;
```

That reads kind of strange with `dockerfile files`.

---

It is the official headline of the upstream repository. I think they thought about that. So left it as-is.